components/automate-chef-io/content/docs/default-policies.md

```diff
@@ -61,6 +62,7 @@ JobScheduler | GetStatusJobScheduler | /retention/nodes/status | GET | cfgmgmt:s
 JobScheduler | ConfigureNodesMissingScheduler | /retention/nodes/missing-nodes/config | POST | cfgmgmt:scheduler:missing-node | configure
 JobScheduler | ConfigureDeleteNodesScheduler | /retention/nodes/delete-nodes/config | POST | cfgmgmt:scheduler:delete-node | configure
 JobScheduler | ConfigureMissingNodesForDeletionScheduler | /retention/nodes/missing-nodes-deletion/config | POST | cfgmgmt:scheduler:missing-nodes-deletion | configure
+{{% /table %}}
```

Comment:
{{% table %}} before and {{% /table %}} after the data content of the table will add the table shortcode. If you would rather this be named something such as `responsive-table` or `table-container` let me know.

Reply:
@IanMadd Look at this shortcode!
Thanks @Shadae! I think calling it responsive-table would be helpful--at least as a reminder for ourselves and info for future docs people why we used it.